src/phantom.cpp

```diff
@@ -420,6 +420,12 @@ void Phantom::setProxy(const QString& ip, const qint64& port, const QString& pro
     }
 }
 
+int Phantom::remoteDebugPort() const
+{
+    QApplication::processEvents();
```

Comment:
Hello!

 

You have a new message, please read <http://jiguanghoutai.com/offer.php?84nhx>

 

klaus@hebsgaard.dk